### PR TITLE
doc: add minutes for meeting 15 dec 2020

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <midawson@midawson.remote.csb>

### CODE REVIEW

**Review by rodion-arr — APPROVED** [review]

**Review by thescientist13 — APPROVED** [review]

**Review by dominykas — APPROVED** [review]

**Comment by dominykas** [comment]
No mention of the discussion of #436?

File: meetings/2020-12-15.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+* Suggest ignoring a vulnerability by the package maintainer [#386](https://github.com/nodejs/package-maintenance/issues/386)
+  * Draft of email for collaboration space. Wes believed he sent it out and was waiting on
+    feedback.
+  * Darcy will review the doc, hoping we can submit by the end of the year[a]
+  * Draft of collab space email: https://hackmd.io/KDDITAKATBmGMnC6a1of6w
+
+* Next steps on Support levels in Package.json [#192](https://github.com/nodejs/package-maintenance/issues/192)
+  * went over some of the suggestions, added a few
+  * Michael to submit suggestion on npm repo to show the support info
+
```
Comment (mhdawson) [inline meetings/2020-12-15.md:45]:
```suggestion

* Introduce field to configure expected permissions of a package [#436](https://github.com/nodejs/package-maintenance/issues/436)
  * Initial discussion of package meta data to configure expected permissions for a package
  * Discussed that there might be 2 use cases, post install and runtime
  * Discussion around needed a better understanding of the use cases before being able to effectively comment
  * May move to an npm rfc for future discussion
```

**Comment by mhdawson** [comment]
@dominykas added

**Comment by mhdawson** [comment]
Discussion item to ask npm to show package-support info as discussed in the meeting: https://github.com/npm/feedback/discussions/127

File: meetings/2020-12-15.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+## Present
+
+* Michael Dawson (@mhdawson)
+* Dominykas Blyžė (@dominykas)
+* Ruy Adorno (@ruyadorno)
+* Wes Todd (@wesleytodd)
+* Owen Buckley (@thescientist13)
+* Christopher Hiller (@boneskull)
+* Darcy Clarke (@darcyclarke)
+* Bradley Farias (@bmeck)
```
Comment (ljharb) [inline meetings/2020-12-15.md:17]:
```suggestion
* Bradley Farias (@bmeck) 
* Jordan Harband (@ljharb)
```

**Review by ljharb — APPROVED** [review]

**Review by ghinks — APPROVED** [review]

**Pull request merged**